packages/renderer/src/lib/pod/PodStatusIcon.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+import PodIcon from '../images/PodIcon.svelte';
+import StarIcon from '../images/StarIcon.svelte';
+
+export let status = '';
+</script>
+
+<div class="grid place-content-center" style="position:relative">
+  <div
+    class="grid place-content-center w-7 h-7 rounded"
+    class:bg-started="{status === 'RUNNING'}"
+    class:border-stopped="{status === 'CREATED' || status === 'EXITED'}"
+    class:text-stopped="{status === 'CREATED' || status === 'EXITED'}"
+    class:border-degraded="{status === 'DEGRADED'}"
+    class:text-degraded="{status === 'DEGRADED'}"
+    class:border-2="{status === 'DEGRADED' || status === 'CREATED' || status === 'EXITED'}"
+    title="{status}">
+    <PodIcon size="20" solid="{status === 'RUNNING'}" />
+  </div>
+  {#if status === 'CREATED'}
+    <StarIcon size="8" style="position:absolute;top:0;right:0" />
+  {/if}
```

Comment:
From what I see the PodStatusIcon is not really associated to a pod but to a status for the color and we provide a specific icon there for created.

It could reuse instead a more generic StatusIcon component where you provide different set of icons (like default icon) or specific icon for a given state.
Then you could achieve your goal of having a common set of colors for stuff related to a status (or mapping status to other states)